source/_integrations/nextcloud.markdown

```diff
@@ -0,0 +1,46 @@
+---
+title: Nextcloud Sensor
+description: Instructions on how to integrate Nextcloud monitor api data into home-assistant.
+logo: nextcloud.png
```

Comment:
Missing the logo file

Reply:
@klaasnicolaas Thanks! I've added the logo file - I'm not sure if it needs to be resized or not.